### PR TITLE
new

### PR DESCRIPTION
**Pull request by darmiel**

### CODE REVIEW

File: program.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package main
+
+import "fmt"
+
+func main() {
```
Comment (statzie) [inline program.go:5]: hello!

File: program.go [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package main
+
+import "fmt"
+
+func main() {
+	fmt.Println("Hello woorld!")
```
Comment (statzie) [inline program.go]:
```suggestion
	fmt.Println("Hello world!")

```

File: program.go
Comment (statzie) [inline program.go:1]: rename to `Program.go`

**Review by statzie — CHANGES_REQUESTED** [review]
please change some things.

File: program.go (hunk 1 above)
Comment (darmiel) [reply program.go:5]: hi!

File: program.go
Comment (darmiel) [reply program.go:1]: done! :)

**Pull request closed without merging**